### DISCUSSION TITLE
Card

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7080] [Help] [answered]
Is there a solution to make the whole card clickable, not just the title ?

**Comment by R2bEEaton** [comment, marked as answer]
Yes. Use "Click Action" with "Navigate To" instead of the "Link URL" option.
This came out in a recent update, so make sure if you update Budibase (if self hosted) and update your app to the latest version.
(See #6788)

![image](https://user-images.githubusercontent.com/34921506/182822825-87dcf675-895c-49e8-b1f0-ab1904f761cb.png)

Enjoy!

**Comment by rafikApp** [comment]
thank you @R2bEEaton. but it didn't work with cards block

**Reply by R2bEEaton** [reply]
I see, you are trying to do this with the Cards *Block* rather than an individual Card element.

It looks like there's no way to do that right now. Instead, you can make a Repeater Block (or regular repeater), and inside that put a normal Card. Then, set the data provider for the repeater, and set the different elements of the card with `{{ New Repeater Block.YourData.YourColumn }}` for example.

And for future reference, you can reply to a Github comment thread in the "Write a reply" box.

**Reply by rafikApp** [reply]
Yes i do that, but i still have a problem

I have a card which is composed of a title and a button which have different actions, I want to trigger the action of the title whatever I click on the title or the empty part of the card (because I'm going to use the app on a phone, otherwise it will be difficult to click exactly on the title using a phone)

![Capture](https://user-images.githubusercontent.com/106697208/182896404-ac03eea0-2995-4525-be5f-acae9377a00e.PNG)

**Reply by R2bEEaton** [reply]
Ok, it's not currently possible to do this - and I can understand why. If you missed the button by even a millimeter, you would activate the wrong action. It's either all or nothing from a design standpoint. You will have to stick with the clickable title and button OR change to have the entire thing clickable (and remove the button).

If it was me, I would make the entire card clickable then have it bring you to a different screen, then on that screen have the "Add intervention" button. There are other ways to do it, this is just one, but basically what you want specifically does not exist for good reasons.

**Reply by rafikApp** [reply]
Ok thank you @R2bEEaton
